support-frontend/assets/components/subscriptionCheckouts/address/addressFields.tsx

```diff
@@ -56,6 +56,13 @@ const marginBottom = css`
 	margin-bottom: ${space[6]}px;
 `;
 
+const selectStateStyles = css`
+	&:invalid:not(&:user-invalid) {
+		/* Remove styling of invalid select element */
+		border: 1px solid ${palette.neutral[46]};
+	}
+`;
+
```

Comment:
We do this [here](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/components/personalDetails/stateSelect.tsx#L60-L72) too.

Maybe worth creating a `Select` of our own applying this - but we should also fix this upstream in Source.

Reply:
Nice, I've never used this `:user-invalid` pseudo class before

Reply:
It's fairly new. It's not [compatible](https://developer.mozilla.org/en-US/docs/Web/CSS/:user-invalid#browser_compatibility) with older browsers but that's probably okay since it's not preventing any usability